torch/_higher_order_ops/_invoke_quant.py

```diff
@@ -12,10 +12,10 @@ class InvokeQuantTracer(BaseHOP):
     def __init__(self) -> None:
         super().__init__("invoke_quant_packed")
 
-    def __call__(self, subgraph, operands, *, scheme=None, quant_options=None):
+    def __call__(self, subgraph, *operands, scheme=None, quant_options=None):
```

Comment:
@eellison I didn't invoke_quant_packed in this PR for reasons:
- it takes in quant_options=None but invoke_quant doesn't accept quant_options
- deleting means we would need to rewrite the pass that moves the quant_options to the node.meta field and there were already a number of moving pieces in this PR

Reply:
I can volunteer to resolve it in a followup with a design of your choosing (I can give invoke_quant the quant_options argument but note that it will be None during inductor because the quant_options will be on the node.meta field)